tests/step_implementers/container_image_static_compliance_scan/test_openscap_compliance.py

```diff
@@ -1,464 +1,92 @@
-            result4.stderr=b'stderr'
-            result4.stdout=b'mount path'
-            result4.returncode=123
-
-            buildah_mock.side_effect = [result1,
-                                        result2,
-                                        result3,
-                                        result4]
-
-            with self.assertRaisesRegex(
-                    ValueError,
-                    r"Zero vfs base containers should be running"):
-                run_step_test_with_result_validation(temp_dir, 'container-image-static-compliance-scan', config, {})
-
-            buildah_mock.assert_called
-
-    @patch('sh.buildah', create=True)
-    def test_container_image_static_compliance_scan_specify_openscap_implementer_buildah_error(self, buildah_mock):
-
-        git_tag = 'git_tag'
-
-        with TempDirectory() as temp_dir:
-            temp_dir.makedir('tssc-results')
-
-            image_tar_file_path = f'{temp_dir.path}//image.tar'
-            temp_dir.write(
-                'tssc-results/tssc-results.yml',
-                bytes(
-                    f'''tssc-results:
-                  tag-source:
-                    tag: {git_tag}
-                  create-container-image:
-                    image-tar-file: {image_tar_file_path}
-                ''',
-                'utf-8')
-                )
-
-            config = {
-                'tssc-config': {
-                    'global-defaults': {
-                        'application-name': 'ref-app-quarkus',
-                        'service-name': 'backend',
-                        'organization': 'tssc-team'
-                    },
-                    'container-image-static-compliance-scan': {
-                        'implementer': 'OpenSCAP',
-                        'config': {
-                           'destination-url': 'quay-quay-enterprise.apps.tssc.rht-set.com',
-                           'log-level': 'info',
-                           'scap-input-file': '/usr/share/xml/scap/ssg/content/ssg-jre-oval.xml',
-                           'scan-output-absolute-path': '/tmp/scap_output.txt',
-                           'scan-report-absolute-path': '/tmp/scap_compliance_report.html'
-                        }
-                    }
-                }
-            }
-
-            result1 = MagicMock()
-            result1.stdout=b"test"
-            buildah_mock.side_effect = [result1, sh.ErrorReturnCode('buildah', b'stdout', b'stderror')]
-
-            with self.assertRaisesRegex(
-                    RuntimeError,
-                    r"Unexpected runtime error"):
-                run_step_test_with_result_validation(temp_dir, 'container-image-static-compliance-scan', config, {})
-
-    @patch('sh.buildah', create=True)
-    def test_container_image_static_compliance_scan_specify_openscap_implementer_no_version(
+class TestStepImplementeContainerImageStaticVulnerabilityScanOpenSCAP(TestStepImplementeSharedOpenSCAPGeneric):
```

Comment:
Oh god this class name HAS to be shortened. This is painful

Reply:
i dont see a sugestion? why does it matter. its a test class and makes it easy to understand in the test logs...